themes/tiddlywiki/vanilla/base.tid

```diff
@@ -680,7 +680,7 @@ button.tc-untagged-label {
 	vertical-align: middle;
 }
 
-.tc-edit-tags .tc-block-dropdown {
+.tc-edit-tags .tc-block-dropdown, .tc-tag-manager-table .tc-drop-down {
```

Comment:
If a CSS rule is valid for several elements, 2 lines should be used like: 

```
.tc-edit-tags .tc-block-dropdown, 
.tc-tag-manager-table .tc-drop-down {
..
}
```
which makes it much more obvious, what's going on.

Reply:
thanks @pmario, I've changed it